src/adaptation/icap/ModXact.cc

```diff
@@ -244,31 +244,26 @@ void Adaptation::Icap::ModXact::writeMore()
         return;
 
     switch (state.writing) {
-
     case State::writingInit:    // waiting for service OPTIONS
         Must(state.serviceWaiting);
-
+        return;
     case State::writingConnect: // waiting for the connection to establish
-
+        return;
     case State::writingHeaders: // waiting for the headers to be written
-
+        return;
     case State::writingPaused:  // waiting for the ICAP server response
-
+        return;
     case State::writingReallyDone: // nothing more to write
         return;
-
     case State::writingAlmostDone: // was waiting for the last write
         stopWriting(false);
         return;
-
```

Comment:
Is this new line removal done by astyle? If not, please keep this and similar new lines. They make the code more readable IMO and, even if you disagree, this PR is not about improving formatting of otherwise unchanged code.

Reply:
Reinstated spaces